index.html

```diff
@@ -197,6 +199,10 @@
 						### Exercise: Let’s set up our HTML
 						Open up your new project in SublimeText and we’ll make our first web page!
 
+						1. Add the document type
+						1. Add an html element
+						1. Nest a head and body element next to each other, inside the html
```

Comment:
Only added here, but often a few high-level text instructions for exercises would be helpful for students.